powerdns/models/ownership.py

```diff
@@ -0,0 +1,67 @@
+"""
+This module provide a new way of ownership/permission to Domains and Records.
+
+The main concept of this is:
+    - all Domains and Records belongs to a Service
+    - Service has owners
+    - Permissions to adding/editing/deleting Domains and Records comes from
+        this ownership
+"""
+
+from enum import Enum
+
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from powerdns.utils import TimeTrackable
+
+
+class ServiceStatus(Enum):
```

Comment:
(i know it's already merged but..) we don't need so many statuses - in fact we only need to know if service is active or not (simple bool field) and leave the logic behind these statuses elsewhere